src/components/RespondentsTable/RespondentsTable.tsx

```diff
@@ -84,6 +85,18 @@ export const RespondentsTable = (): JSX.Element => {
         filterRows(secretId, searchValue) || filterRows(nickname, searchValue),
     );
 
+  useEffect(() => {
+    if (id && baseBreadcrumbs && baseBreadcrumbs.length > 0) {
+      const respondentsCrumb: Breadcrumb[] = [
+        {
+          icon: <Svg id="respondent-outlined" width="13.5" height="15" />,
+          label: t('respondents'),
+        },
+      ];
+      dispatch(breadcrumbs.actions.setBreadcrumbs([...baseBreadcrumbs, ...respondentsCrumb]));
```

Comment:
y can simplify this      dispatch(breadcrumbs.actions.setBreadcrumbs([...baseBreadcrumbs,   {
          icon: <Svg id="respondent-outlined" width="13.5" height="15" />,
          label: t('respondents'),
        }]));

Reply:
done